src/backend/gl/src/command.rs

```diff
@@ -628,3 +634,37 @@ impl command::Buffer<Resources> for CommandBuffer {
                       instances));
     }
 }
+
+impl c::CommandBuffer for CommandBuffer {
+    type SubmitInfo = SubmitInfo;
+    unsafe fn end(&mut self) -> SubmitInfo {
```

Comment:
can we have `fn end(&mut self) -> &SubmitInfo`, perhaps? or anything else that borrows it instead of creating